src/pages/de/blog/online-speisekarte/index.mdx

```diff
@@ -0,0 +1,102 @@
+
+Dies ist aber doch ein recht abstraktes Thema, wenn man es nur kurz anspricht. Also, fangen wir ganz von vorne an...
+
+## Online Speisekarte
+
+Am einfachsten lässt sich eine online Speisekarte so beschreiben: sie entspricht einer traditionellen Speisekarte auf Papier, nur eben online. 
+
+Eine online Speisekarte ist an das verwendete Gerät angepasst (Handy, Tablet, Web usw.), d.h. sie ist leichter zu lesen und einfacher für die Augen.
+
+Des Weiteren bieten einige online Speisekarten auch integrierte Bestell-und Zahlungsfunktionen.
+
+Klingt interessant, oder? Ich erkläre mehr wie diese Funktionen wirken im Laufe des Artikels.
+
+## Das Internet und die Suchemachine
+
+Google ist die grösste, bekannteste und meistgenutzte Suchmaschine der Welt. Wenn Sie also expandieren und mehr Gäste erreichen wollen, sollte Ihre online Speisekarte dort zu finden sein. 
+
+Die grosse Mehrheit der onlinenutzer sieht sich bei der Suche nach einem Restaurant die online Speisekarte an. Denn die meisten Gäste möchten sich ein Bild von der Art der angebotenen Speisen und der Vielfalt des Angebots in Ihrem Betrieb machen.
+
+Wenn Sie Ihren Betrieb nicht gut durch Ihre online Speisekarte repräsentieren, ist die Wahrscheinlichkeit gross, dass sich potenzielle Gäste nicht für Ihr Restaurant interessieren. Aus diesem Grund sind Fotos, Organisation und Speisekartebeschreibungen so wichtig. 
+
+Und all dies kann über ein CMS auf einfache Weise erfolgen. Aber dafür empfehlen wir einen anderen Artikel: [Speisekarte bearbeiten](/de/dokumentation/speisekarten-angebot-bearbeiten).
+
+## Was bringt mir eine online Speisekarte?
+
+Im Alltag kann eine online Speisekarte grosse geschäftliche Möglichkeiten mit sich bringen, die manchmal nicht auf den ersten Blick ersichtlich sind. Deshalb werde ich hier etwas konkreter berichten, was Sie von einer online Speisekarte noch erwarten können und wie Sie von dieser Speisekarte profitieren:
+
+### SEO Strategien mit einer online Speisekarte 
+
+Eine HTML Version einer Speisekarte, die auf Ihrer Website veröffentlicht wird, ist viel effektiver für das Internet als eine PDF Version. Durch dieses Format kann Google Ihre online Speisekarte durch SEO "lesen" und sie zur Verfügung stellen, wenn ein Gast nach bestimmten Begriffen sucht. 
+
+Beispielsweise: "Spanisches Restaurant Tapas" und Ihre online Speisekarte wird empfohlen. 
+
+Mit einer PDF Version einer Speisekarte wird schwieriger sein, eine gute Platzierung in Suchmachinen bzw. Google zu erreichen.
+
+Durch eine HTML Version werden alle in Ihrer online Speisekarte erwähnten Inhalte zu Getränken, Speisen, Desserts, Angeboten, Ihrem Standort, Restaurantnamen usw. durch die von Google angewandte SEO Strategie für die Gäste leichter auffindbar.
+
+In den meisten Fällen wollen die Gäste, die nach einer online Speisekarte suchen, schnell konsumieren und sich von einem Restaurant überraschen lassen. Seien Sie dieses Restaurant und erreichen Sie mit Ihrer SEO Strategie neue Gäste!
```

Comment:
Wieso wollen Gäste die online suchen schnell konsumieren und sich überraschen lassen?

Reply:
In den meisten Fällen wollen die Gäste, die nach einer online Speisekarte suchen, schnell konsumieren, denn sie haben Hunger und wollen sich von einem Restaurant mit der Gerichten überraschen lassen. Passt es besser so?